CMakeLists.txt

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #===============================================================================
 
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.18.2)
```

Comment:
Just curious, how do we decide 3.18.2 as the minimum cmake version? For 2.8.12, I understand it's the default version distributed along with centos7. Thanks.

Reply:
It is the default version in RHEL8.